packages/react-dom/src/client/ReactDOM.js

```diff
@@ -625,7 +625,7 @@ const DOMRenderer = ReactFiberReconciler({
     rootContainerInstance: Container,
   ): boolean {
     setInitialProperties(domElement, type, props, rootContainerInstance);
-    return shouldAutoFocusHostComponent(type, props);
+    return true;
```

Comment:
This is not good because now every host Fiber gets scheduled for an effect during commit. The whole point of returning a boolean is to avoid this, as ideally this should be a rare case.